diff -Nur mxp.orig/Imakefile mxp/Imakefile
--- mxp.orig/Imakefile	Fri Jun  2 17:26:54 2000
+++ mxp/Imakefile	Sun Sep 16 19:33:20 2001
@@ -32,7 +32,7 @@
 SIZE_INC = 1
 
 XCOMM Installation directory
-BINDIR = /usr/local/bin
+XCOMM BINDIR = /usr/local/bin
 
 XCOMM ******************* End of Configuration ***********************
 
@@ -52,7 +52,7 @@
 LOCAL_LIBRARIES = $(XPM_LIBS) $(XLIBS) -lm 
 INCLUDES = -Ilug $(XPM_INCLUDE)
 
-ComplexProgramTarget(mxp)
+ComplexProgramTargetNoMan(mxp)
 
 clean::
 	$(RM) $(L_OBJS)
